scripts/common/prompts.sh

```diff
@@ -126,15 +93,10 @@ function prompt_for_master_address() {
     if [ -n "$KUBERNETES_MASTER_ADDR" ]; then
         return
     fi
-    if ! prompts_can_prompt ; then
-        bail "kubernetes.masterAddress required"
```

Comment:
i do not understand why this is changing

Reply:
Same we are `read -p "Kubernetes master address: " PROMPT_RESULT` instead